warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoFunctionsDescriptor.java

```diff
@@ -353,8 +355,11 @@ public JexlNode toGeoWaveFunction(Set<String> fields) throws Exception {
                 }
             }
             
-            if (wkt != null)
-                return JexlASTHelper.parseJexlQuery("geowave:intersects(" + getFieldParam(fields) + ", '" + wkt + "')");
+            if (wkt != null) {
+                // using dereference of the child node since we do not want the JexlASTScript not the JexlASTReference parent nodes
```

Comment:
```suggestion
                // using dereference of the child node since we do not want the JexlASTScript nor the JexlASTReference parent nodes
```